rclpy/src/rclpy/node.hpp

```diff
@@ -42,6 +42,20 @@ get_node_name(py::capsule pynode);
  */
 const char *
 get_node_namespace(py::capsule pynode);
+
+/// Get a list of parameters for the current node
+/**
+ * On failure, an exception is raised and NULL is returned if:
+ *
+ * Raises ValueError if the argument is not a node handle.
+ * Raises RCLError if the parameters file fails to parse
```

Comment:
@sloretz meta: I think there are quite a few more failure modes.

Reply:
Documented more exceptions in 33c40a34f62ff30c6490a03a31222d40250e0fc6